tests/plan/terraform/main.tf

```diff
@@ -30,6 +30,7 @@ module "eks" {
   bastion_info        = module.infra.bastion
   create_eks_role_arn = module.infra.create_eks_role_arn
   tags                = module.infra.tags
+  monitoring_bucket   = module.infra.monitoring_bucket
```

Comment:
We should create a plan file in `examples/tfvars` to test `private-link=ON`

Reply:
added